contracts/Auction.sol

```diff
@@ -2,75 +2,145 @@
 pragma solidity >=0.4.22 <0.8.0;
 
 contract Auction {
-  constructor() public {
+    // (X) - string Name – name of the object/auctioned item
+    // (X) - string Description – description of the item
+    // (Sort of. See getTimeRemaining function) - Var time_remaining - countdown
+    // (X) - Address owner_address - current owner
+    // (X) - Float price - minimum_price/marked_by_seller
+    // (Return type is different) - Float current_highest_bid - the highest bid at the time
+    // (X) - Address current_highest_bidder - the highest bidder at the time
+    //     - List [] bid_history - bids that have been placed with addresses and bid amount
+    //    - Time&Date Start_time - time of start.
+    // (Return type is different) - Time&Date End_time - time the contract is supposed to end
+    // (same as owner_address right?) - Ethereum address - Auction’s owner’s address
+    // (is basically current_highest_bidder after the auction is over right?) - Ethereum address - Item’s owner address
+    //     - MinimumBidIncrement - minimum amount to bid (set to 1% of the item's current price)
 
-  }
 
 
-   struct item {
-     uint id;
-     string name;
-   }
 
+    address payable public seller_address;
+    uint public End_time;
```

Comment:
shall we use snake case for naming convention ? `end_time` instead of `End_time` so it would be more consistent